xml/System.Collections.Immutable/ImmutableArray`1+Builder.xml

```diff
@@ -646,7 +680,15 @@
         <param name="item">The item to locate in the array.</param>
         <summary>Determines the index of a specific item in the array.</summary>
         <returns>The index of <paramref name="item" /> if it's found in the list; otherwise, -1.</returns>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+If an object occurs multiple times in the list, the <xref:System.Collections.Generic.IList%601.IndexOf%2A> method always returns the first instance found.
```

Comment:
This remark was imported from the interface method implemented by this method. Is it still relevant?